label_weeds.py

```diff
@@ -261,8 +326,9 @@ def run(self):
     import argparse
     parser = argparse.ArgumentParser(description='__doc__')
     parser.add_argument('path', help='Path to image directory.')
-    parser.add_argument('--out', help='Specify an annotation file name (.json)', default="test.json")
+    parser.add_argument('--out', help='Specify an output directory for annotations.', required=True)
+    parser.add_argument('--labels', nargs='+', help='List of label names.')
```

Comment:
Nastavil bych default hodnoty pro labels

Reply:
LabelImg myslím požaduje labels v samostatném texťáku. Nechceš si ten soubor vytvořit dopředu a zde automaticky načítat? Jakákoli ruční práce je zdrojem chyb.

Reply:
Stačí momentálně jen jeden druh na masovou anotaci.